src/Commands/core/ArchiveDumpCommands.php

```diff
@@ -64,7 +64,7 @@ final class ArchiveDumpCommands extends DrushCommands
      */
     #[CLI\Command(name: self::DUMP, aliases: ['ard'])]
     #[CLI\ValidatePhpExtensions(extensions: ['Phar'])]
-    #[CLI\Option(name: 'destination', description: 'The full path and filename in which the archive should be stored. If omitted, it will be saved to the drush-backups directory.')]
+    #[CLI\Option(name: 'destination', description: 'The full path and filename in which the archive should be stored, relative to Drupal root, normally "web" for drupal/recommended-project projects. If omitted, it will be saved to the /tmp directory.')]
```

Comment:
```suggestion
    #[CLI\Option(name: 'destination', description: 'The full path and filename in which the archive should be stored. Any relative path will be calculated from Drupal root (usually <info>web</web> for drupal/recommended-project projects). If omitted, it will be saved to the configured temp directory.')]
```